### PR TITLE
Qstory/qstr 97

### PR DESCRIPTION
**Pull request by rsatyasi**
Reverted the password configuration in pom.xml

### CODE REVIEW

File: QStore4S/pom.xml [hunk 1]
```diff
@@ -200,8 +220,8 @@
 			<!-- Dev Env. Properties -->
 			<properties>
 				<!-- Neo4j Properties -->
-				<neo4j.user>${neo4j.username}</neo4j.user>
-				<neo4j.pw>${neo4j.password}</neo4j.pw>
+				<neo4j.user>neo4j</neo4j.user>
+				<neo4j.pw>password</neo4j.pw>
```
Comment (jdamerow) [inline QStore4S/pom.xml:224]: These need to be variables like the other parameters (e.g. deploy.path). Right now you still hardcode username and password (username = neo4j, password = password).

**Comment by jdamerow** [comment]
see comment

**Comment by rsatyasi** [comment]
You want me to change it to

<neo4j.user>${neo4j.username}</neo4j.user>
<neo4j.pw>${neo4j.password}</neo4j.pw>

?

**Comment by jdamerow** [comment]
Yes. Like it was before (the two red lines).

**Pull request closed without merging**